test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1365,12 +1365,12 @@ func getStatusTimestamp(status string) (time.Time, error) {
 		return time.Time{}, err
 	}
 
-	timestampMatches := timestampMatcher.FindAllStringSubmatch(status, -1)
-	if len(timestampMatches) < 1 {
+	timestampMatch := timestampMatcher.FindStringSubmatch(status)
+	if len(timestampMatch) < 2 {
```

Comment:
@aleksandra-malinowska for my own curiosity, do you mind sharing why you used the `len(timestampMatch) < 2` check instead of `len(timestampMatch) == nil`? My understanding is `FindStringSubmatch` returns nil if no match is found. Thanks :)